config/routes.rb

```diff
@@ -3,14 +3,25 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'task#index'
-  get "/tasks/confirm/:id" => 'task#confirm'
-  get '/tasks/destroy/:id' => 'task#destroy'
+  root 'tasks#index'
 
-  get '/tasks/new' => 'task#new'
-  post '/tasks' => 'task#create'
+  get '/tasks/confirm/:id' => 'tasks#confirm'
+  get '/tasks/destroy/:id' => 'tasks#destroy'
 
```

Comment:
To make your routes more RESTful you'd want the ID in between the controller and action, i.e. /tasks/:id/confirm, /tasks/:id/destroy, etc.